portfolio/src/main/webapp/community.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Community Page</title>
+    <script src="community.js"></script>
+    <link rel="stylesheet" href="community.css">
+  </head>
+  <body onChange="renderComments()" onload="renderComments()">
+    <div id="community-content">
+        <div id="comment-form-section">
+            <div id="header">
+                <a href="index.html"> <img src=/images/profile.jpg /> </a>
+                <div class="left">
+                    <a href="about.html"> <p> About </p> </a>
+                    <a href="survey.html"> <p> Survey </p> </a>
+                    <a href="#"> <p> Contact </p> </a>
+                </div>
+            </div>
+            <h2>Here are all the wonderful things people have to say about this website</h2>
+            <form action="/comments" method="POST">
+                <p>Username:</p>
```

Comment:
I think it's better practice to use the "label" element instead of "p" for text describing inputs in a form, since it creates a stronger programmatic association between the text on the screen and the input field, which among other things can be a big help when it comes to accessibility features for people with disabilities.

A little bit more about the "label" element here:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

Reply:
Changed to label